Jenkinsfile

```diff
@@ -0,0 +1,64 @@
+pipeline {
+  agent any
+  tools {
+        maven 'Maven3'
+        jdk 'Java8'
+  }
+  options {
+    buildDiscarder logRotator(numToKeepStr: '10')
+  }
+  stages {
+    stage('Clean') {
+      steps {
+        sh 'mvn clean'
+      }
+    }
+    stage('Version') {
+      steps {
+        sh 'mvn versions:set -DnewVersion=2.1.16.1'
```

Comment:
This version doesn't look right.
I'm pretty sure we can skip this entire stage.